deploy/spinnaker/basic/spinnakerservice.yml

```diff
@@ -9,12 +9,10 @@ spec:
       persistentStorage:
         persistentStoreType: s3
         s3:
-          bucket: mybucket # Change me
+          bucket: spinnaker
```

Comment:
`spinnaker` sounds like a system thing that I shouldn't change. Can we either leave a note to change it or make it more obvious it's an example? Not everyone will know what this setting represents.

Reply:
I'll leave a note that it can be changed safely. The idea here is to provide sensible defaults to avoid having users to change many settings to get started, just like we're creating the namespace `spinnaker-operator` and set it as default instead of having them to go an update the manifest to put a desired namespace.

Reply:
`spinnaker` won't work because s3 buckets are globally (like real real globally) unique

Reply:
how about `my-spinnaker-storage-bucket`? to make it more obvious? or `CHANGEMEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE`